aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1239,7 +1239,7 @@ impl AptosVM {
         change_set: &VMChangeSet,
     ) -> Result<(), VMStatus> {
         assert!(
-            change_set.aggregator_write_set().is_empty(),
+            change_set.aggregator_v1_write_set().is_empty(),
```

Comment:
@zekun000 @georgemitenkov 
random question: what's the waypoint change set? should delta set also be empty? (is there gas charged?)

Reply:
I think @zekun000 should know better, but from my understanding when we want to apply a write set to the chain (e.g. a validator comes online and doesn't want to re-execute transactions and instead applies a write set for faster state sync) we use this type of transaction, not sure how often and how read-before-write is crucial here?

Reply:
waypoint writeset is for disaster recovery scenarios like losing quorums where we need social consensus for all operators to manually apply some changes on top of db to resume the network